pkg/pillar/hypervisor/kvm.go

```diff
@@ -108,8 +108,8 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
 
 [smp-opts]
   cpus = "{{.VCpus}}"
-  sockets = "{{.VCpus}}"
-  cores = "1"
+  sockets = "1"
```

Comment:
Is sockets something which need to match the hardware? Or you can just pretent that all of the cores are in a single CPU socket?

Reply:
since all these are QEMU virtual processor instances, we can pretend that there is a single socket with x number of cores.